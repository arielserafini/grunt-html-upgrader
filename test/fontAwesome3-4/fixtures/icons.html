<a><i class="icon-remove icon-large icon-spin"></i></a>
<a><i class="icon-ban-circle"></i></a>
