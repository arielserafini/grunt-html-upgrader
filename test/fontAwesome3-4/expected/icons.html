<a><i class="fa fa-times fa-large fa-spin"></i></a>
<a><i class="fa fa-ban"></i></a>
